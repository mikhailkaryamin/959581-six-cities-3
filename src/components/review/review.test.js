import React from "react";
import renderer from "react-test-renderer";
import Review from "./review.jsx";

const review = {
  id: 1,
  user: {
    id: 1,
    isPro: true,
    name: `Mi`,
    avatarURL: `img/avatar-max.jpg`
  },
  rating: 2,
  comment: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  Curabitur ornare nunc a blandit ultricies.
  Integer a urna in nunc congue efficitur.`,
  date: new Date(2010, 10, 1)
};

it(`Render correctly review`, () => {
  const tree = renderer
    .create(
        <Review
          review={review}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});

