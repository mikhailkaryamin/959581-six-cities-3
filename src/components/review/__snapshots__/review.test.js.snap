// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly review 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="img/avatar-max.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Mi
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "40%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  Curabitur ornare nunc a blandit ultricies.
  Integer a urna in nunc congue efficitur.
    </p>
    <time
      className="reviews__time"
      dateTime="2010-10-1"
    >
      November 1, 2010
    </time>
  </div>
</li>
`;
